include/z64camera.h

```diff
@@ -16,25 +16,73 @@
 
 #define SUB_CAM_ID_DONE 0 // Used in some actors for variables holding sub camera IDs to indicate "subcam is finished"
 
+#define CAM_HUD_VISIBILITY_SHIFT 8
+#define CAM_HUD_VISIBILITY_MASK (0x0F00)
+#define CAM_HUD_VISIBILITY(hudVisibility) (((hudVisibility) & 0xF) << CAM_HUD_VISIBILITY_SHIFT)
+
+#define CAM_HUD_VISIBILITY_50 CAM_HUD_VISIBILITY(0)
```

Comment:
Why 50 and not 0?

Reply:
This can be seen from this PR, and the `CAM_HUD_VISIBILITY` macros will be adjusted once that PR is in:
https://github.com/zeldaret/mm/pull/942

The idea is that the hud visibility is bitpacked into a single nibble, so it only has the range `0x0 - 0xF`. However, the actual hud-visibility values use a magic number: 50, to represent "turn the hud back on".

So camera's `0` maps to interfaces `50`.

Once that other PR is in though, it will be named to sync with that PR, so it's only a temporary name to help with mapping until the hud-visibility PR is into master